src/farm.ts

```diff
@@ -0,0 +1,27 @@
+import fs from 'node:fs'
+import { plugin, type Options } from './core'
+import type { JsPlugin, PluginLoadHookParam } from '@farmfe/core'
+
+const createFarmVuePlugins = (options?: Options): JsPlugin[] => {
```

Comment:
Since the farm does not load other types of files by default, it is necessary to wrap it with a plugin in advance to ensure normal operation without affecting other frameworks.